postgresql-persistence/src/main/java/com/netflix/conductor/aurora/AuroraExecutionDAO.java

```diff
@@ -248,10 +251,25 @@ public Task getTask(String taskId) {
 	@Override
 	public Task getTask(String workflowId, String taskRefName) {
 		String GET_TASK = "SELECT json_data FROM task WHERE workflow_id = ? and task_refname = ? ORDER BY id DESC";
-		return queryWithTransaction(GET_TASK, q -> q
-			.addParameter(workflowId)
-			.addParameter(taskRefName)
-			.executeAndFetchFirst(Task.class));
+		return queryWithTransaction(GET_TASK, q->{
+					ResultSet rs = q.addParameter(workflowId)
```

Comment:
As I said in Slack, would make sense to create Result Set Handler which accepts result set and produces Task. We need to call this RS handler couple of times.